include/groonga/error.h

```diff
@@ -33,3 +33,18 @@ grn_windows_error_code_to_rc(int error_code);
 #ifdef __cplusplus
 }
 #endif
+
+/*! \page error_debugging Error Debugging
+ *
+ *  \section error_details Additional Error Details
+ *
+ *  For more detailed error information, check the `ctx->rc`(\ref _grn_ctx::rc)
```

Comment:
Can we use `grn_ctx`?

```suggestion
 *  For more detailed error information, check the `ctx->rc`(\ref grn_ctx::rc)
```